### ISSUE TITLE
Create Q&A or something related web page for "ELI5" (explain like I'm 5) topics

### ISSUE THREAD

**Issue by wiardvanrij** [thanos-io/thanos#4091]
### Is your proposal related to a problem?

I personally find it hard to comprehend certain moving parts in Thanos. I.e blocks, compacting, chunks. Horizontal vs vertical. I'm aware of pages like https://thanos.io/tip/components/compact.md/#vertical-compactions - They are quite rich in information. However you do need knowledge of multiple other parts to understand it.
I noticed the same from other users. For example, we recently had a question: how should I calculate the chunk-pool-size? Honestly, I also don't know.
So, I want to ask for some ELI5 (explain like I'm 5) parts. It doesn't have to be in proposals, but perhaps proposals can have links to such explanation in the docs/website.
Benefits are that devs can tag along faster, users gain more knowledge and perhaps proposals gain more input.

### Describe the solution you'd like

Q&A page
I made a PR as example here: https://github.com/thanos-io/thanos/pull/4090

### Describe alternatives you've considered

I thought about it, but personally I couldn't think about any. Open for it though!

**Comment by bwplotka** [comment]
We talked about this for a long time, I even had some FAQ started at some point. The main issue is that most of FAQ questions should be addressed by docs. Yet I think practice shows that we should have some duplication of docs and links to even better experience. Thanks for starting this! (:

**Comment by sevagh** [comment]
If you're 5 years old, you shoudn't be running production distributed metrics systems.
